dockerfiles/rbe-ubuntu22-04/Dockerfile

```diff
@@ -0,0 +1,121 @@
+FROM marketplace.gcr.io/google/ubuntu2204@sha256:b70be2d80535d3fe99e2bc484133da588728add8fc15956d124c478b42e61f2c
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Python 2 and 3
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      python2 \
+      python2-dev \
```

Comment:
wdyt about removing python2 and then only installing it later if it turns out to be needed?

```suggestion
# Python 3
RUN apt-get update && \
    apt-get install -y --no-install-recommends \
```